public/js/common/objects/react_course.js

```diff
@@ -0,0 +1,100 @@
+var Course = React.createClass({
+
+    getDefaultProps: function() {
+        var course = getCourse("CSC108H1");
```

Comment:
I originally passed the course name in from outside the component, but now it seems like `this.props` doesn't work in this function